Add start scripts for checker and rule manager (#230)
guardian/typerighter: docs/01-running-locally.md

# Basic developer run

Get credentials for Composer profile.

> To launch the Checker, run `./script/run-checker` (`--debug` to run a debugger on port 5005).

Reviewer: ```suggestion
To launch the Checker, run `./script/start-checker` (`--debug` to run a debugger on port 5005).
```

> To start the Rule Manager, run `./script/run-manager` (`--debug` to attach a debugger on port 5006) and visit `https://manager.typerighter.local.dev-gutools.co.uk/` to check it is running.

Reviewer: ```suggestion
To start the Rule Manager, run `./script/start-manager` (`--debug` to attach a debugger on port 5006) and visit `https://manager.typerighter.local.dev-gutools.co.uk/` to check it is running.
```

Author: ah thanks for spotting this!

> To start the Rule Manager, run `./script/start-manager` (`--debug` to attach a debugger on port 5006) and visit `https://manager.typerighter.local.dev-gutools.co.uk/` to check it is running.

Reviewer: might be nice if this were a link to save some keystrokes?
```suggestion
To start the Rule Manager, run `./script/start-manager` (`--debug` to attach a debugger on port 5006) and visit [the locally running app](https://manager.typerighter.local.dev-gutools.co.uk) to check it is running.
```

> To run everything, run `./script/start` (`--debug` will attach debuggers on both services in the respective ports above) and visit `localhost:9000` to confirm the service came up correctly.

Reviewer: ```suggestion
To run everything, run `./script/start` (`--debug` will attach debuggers on both services in the respective ports above) and visit [the locally running app](https://manager.typerighter.local.dev-gutools.co.uk/) to confirm the service came up correctly.
```

Author: thank you!
